db-dump-entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env sh
+# Alternative entrypoint for the inmanta orchestrator container.
+# Instead of starting the orchestrator, it starts a cron daemon, to allow
+# pg_dump to periodically run and export a database dump.
+
+set -x
+set -e
+
+# Install cron, logrotate and tini
+apt-get update
+apt-get install -y cron logrotate  tini
+
+# Configure db dump script, save container env vars in script to simplify
+# container configuration
+if [ ! -f /db_dump ]; then
+    cat > /db_dump <<EOF
+#!/usr/bin/env sh
+$(export)
```

Comment:
Why `export`?

Reply:
Because `export` without any arguments, prints to stdout the `export` commands required to set the environment variables which are currently set